### DISCUSSION TITLE
Operations menu fixed position

### DISCUSSION THREAD

**Discussion by mjazwinski** [Azure/api-management-developer-portal#2577] [Feature request]
It would be great if operations menu in operation details widget could be fixed so that it is always visible. Then it case of larger documents it would stay visible.

![menu_azure_apim_dev_portal](https://user-images.githubusercontent.com/3313203/119841356-03d60a00-bf06-11eb-9497-d1ffa9afeff7.png)

**Comment by ghost** [comment]
@mjazwinski, thank you for opening this issue. We will triage it within the next few business days.

**Comment by mikebudzynski** [comment]
@mjazwinski, what do you mean by "fixed" and always visible? Can you provide reproduction steps and clarify the expected behavior?

**Comment by ghost** [comment]
@mjazwinski, we need more information before we start working on this issue. If you prefer to share it in private, please send us an email to apimportalfeedback@microsoft.com with the issue number in its subject.

**Comment by AnRei123** [comment]
We are interested in a kind of floating panel that always allows you to scroll through and navigate the list of operations irrespective how long the API details page is. Remark: depending on the kind of operations our API details pages are VERY long and you need to scroll a lot. But the user should be able to navigate to another operation without the need to scroll to the top of the page first.

**Comment by mjazwinski** [comment]
@mikebudzynski the operations list should stay visible in the viewport when you scroll the page. See this examples:
- https://www.w3.org/Style/Examples/007/menus.en.html
- https://www.w3schools.com/howto/howto_css_fixed_menu.asp
Like @AnRei123  wrote if API has data that makes the displayed page scroll then in order to change operation or see other operations you constantly have to scroll up

**Comment by ghost** [comment]
@mjazwinski, thank you for requesting this feature.
